mypyc/test-data/run-dicts.test

```diff
@@ -92,10 +92,14 @@ assert get_content(od) == ([3, 1], [4, 2], [(3, 4), (1, 2)])
 assert get_content_set({1: 2}) == ({1}, {2}, {(1, 2)})
 assert get_content_set(od) == ({1, 3}, {2, 4}, {(1, 2), (3, 4)})
 
+from collections import defaultdict
 d = {}
 assert d.copy() == d
 d = {'a': 1, 'b': 2}
 assert d.copy() == d
+assert d.copy() is not d
+d = defaultdict(int)
+assert d.copy() == d
```

Comment:
Use a non-empty default dict instance, and check that the type of the return value is correct (`assert isinstance(d.copy(), defaultdict)`.